introduction.md

```diff
@@ -3,3 +3,6 @@ This is the introduction to Tanner Crockett
 live in Salt Lake City Utah
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?